patch/kconfig-inclusions

```diff
@@ -124,6 +124,14 @@ CONFIG_SPI_INTEL_PLATFORM=m
 CONFIG_SPI_INTEL=m
 # For pensando
 CONFIG_IONIC=m
+# For AMD Ryzen Embedded processors
+CONFIG_PCI_ENDPOINT=y
+CONFIG_PCIE_DW=y
+CONFIG_PCIE_DW_HOST=y
+CONFIG_PCIE_DW_EP=y
+CONFIG_PCIE_DW_PLAT=y
+CONFIG_PCIE_DW_PLAT_HOST=y
+CONFIG_PCIE_DW_PLAT_EP=y
```

Comment:
can we use m instead of y?

Reply:
These Kconfig options of the kernel configuration are set to 'bool' and not 'tristate'.